packages/calcite-components/src/controllers/useCancelable.ts

```diff
@@ -0,0 +1,47 @@
+import { makeGenericController } from "@arcgis/lumina/controllers";
+import type { Arrayable } from "type-fest";
+import { LitElement } from "@arcgis/lumina";
+import { Cancelable } from "../tests/commonTests/interfaces";
+
+/**
+ * Interface for the CancelableController.
+ */
+export interface useCancelable {
```

Comment:
Make sure interfaces and classes use `PascalCase`. The controller export should keep camelcase.